state-sync/state-sync-driver/src/tests/mocks.rs

```diff
@@ -140,34 +140,34 @@ mock! {
             txn_list_with_proof: TransactionListWithProof,
             verified_target_li: &LedgerInfoWithSignatures,
             epoch_change_li: Option<&'a LedgerInfoWithSignatures>,
-        ) -> Result<()>;
+        ) -> AnyhowResult<()>;
```

Comment:
`anyhow::Result` is not too much longer, and I think explicitly say DbResult is better than this.